frontend/src/pages/warehouse/ModalAddWarehouse.js

```diff
@@ -0,0 +1,324 @@
+import React, {useState} from 'react';
+import Modal from "react-bootstrap/Modal";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import ErrorMessage from "../../messages/errorMessage";
+import {AsyncTypeahead} from "react-bootstrap-typeahead";
+import Dropdown from "react-bootstrap/Dropdown";
+import validateWarehouse from "../../validation/WarehouseValidationRules";
+import {FaTrash} from "react-icons/fa";
+
+function ModalAddWarehouse(props) {
+
+    const ref = React.createRef();
+    const [stateOptions, setStateOptions] = useState([]);
+    const [dispatcherOptions, setDispatcherOptions] = useState([]);
+    const [dispatchersList, setDispatchersList] = useState([]);
```

Comment:
Rename dispatchersList to dispatchers. Take it as a general rule for list variables naming.

Reply:
thanks